docs/axisman.rst

```diff
@@ -72,14 +72,22 @@ hold other AxisManagers, and some other stuff too).
 
     dset = core.AxisManager().wrap('tod', tod)
 
+AxisManagers can also hold scalars (ints, floats, strings, etc.), note that ``None`` is considered a scalar as well.
+
+.. code-block:: python
+
+    dset = dset.wrap('scalar', 1.0)
+
 Inspecting::
 
     >>> print(dset)
-    AxisManager(tod[3,10000])
+    AxisManager(tod[3,10000], scalar[0])
```

Comment:
The notation scalar[0] implies that scalar is 1-d array with shape (0,).  I think the repr in this case should just be
```
 AxisManager(tod[3,10000], scalar)
```